### PR TITLE
Fix cortex query seconds total

### PR DESCRIPTION
**Pull request by Wing924**
**What this PR does**:
bugfix: Query-Frontend: `cortex_query_seconds_total` now return seconds not nanoseconds.

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by Wing924** [comment]
before:
スクリーンショット 2020-12-11 17 58 57
after:
スクリーンショット 2020-12-11 19 17 35

**Review by pstibrany — APPROVED** [review]
Good catch, Thanks!

**Comment by pstibrany** [comment]
@jtlisi Worth including in Cortex 1.6.0?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT] Memberlist: add status page with available details about memberlist-based KV store and memberlist cluster. It's also possible to view KV values in Go struct or JSON format, or download for inspection. #3575
```
Comment (pracucci) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by pracucci — APPROVED** [review]
Thanks for the fix!

**Comment by jtlisi** [comment]
> @jtlisi Worth including in Cortex 1.6.0?

I think so. I'll make a note to cherry pick this for `rc.1`

**Review by jtlisi — APPROVED** [review]
LGTM

**Pull request merged**